src/future-hopr-lib-components/Modal/styled.tsx

```diff
@@ -1,11 +1,19 @@
 import styled from '@emotion/styled';
 import { Dialog, DialogContent, IconButton } from '@mui/material';
 
-export const SDialog = styled(Dialog)`
+export const SDialog = styled(({
+  width,
+  ...rest
+}: any) => <Dialog {...rest} />)`
```

Comment:
We should change this any in either this pr or another, the code does work but we should try to minimize the use of anys

Reply:
updated